### PR TITLE
Update what's new with DSDay blog post

### PR DESCRIPTION
**Pull request by calvin-lau-sig7**

### CODE REVIEW

File: views/partials/_whats-new.njk [hunk 1]
```diff
@@ -6,7 +6,8 @@
     <div class="govuk-grid-row">
       <div class="govuk-grid-column-two-thirds-from-desktop">
         <h2 id="whats-new" class="govuk-heading-l">What’s new</h2>
-        <p class="govuk-body">9 August 2022: We’ve released GOV.UK Frontend v4.3.0. This feature release includes several ‘quality of life’ improvements to make GOV.UK Frontend work better for service teams and meet their needs.</p>
+<p class="govuk-body">16 August 2022: We're running our flagship online event, Design System Day 2022, on 20 and 21 September. <a href="https://designnotes.blog.gov.uk/2022/08/16/join-us-for-design-system-day-2022/" class="govuk-link">See what we have planned and find out how to join us</a><br/></p>
```
Comment (calvin-lau-sig7) [inline views/partials/_whats-new.njk:9]: Should I use a line break or empty `<p>` tag to get a blank line?

File: views/partials/_whats-new.njk [hunk 2]
```diff
@@ -6,7 +6,9 @@
     <div class="govuk-grid-row">
       <div class="govuk-grid-column-two-thirds-from-desktop">
         <h2 id="whats-new" class="govuk-heading-l">What’s new</h2>
-        <p class="govuk-body">9 August 2022: We’ve released GOV.UK Frontend v4.3.0. This feature release includes several ‘quality of life’ improvements to make GOV.UK Frontend work better for service teams and meet their needs.</p>
+<p class="govuk-body">16 August 2022: We're running our flagship online event, Design System Day 2022, on 20 and 21 September.</p>
+<p class="govuk-body"><a href="https://designnotes.blog.gov.uk/2022/08/16/join-us-for-design-system-day-2022/" class="govuk-link">See what we have planned and find out how to join us</a></p>
```
Comment (calvin-lau-sig7) [inline views/partials/_whats-new.njk:10]:
```suggestion
<p class="govuk-body"><a href="https://designnotes.blog.gov.uk/2022/08/16/join-us-for-design-system-day-2022/" class="govuk-link">See what we have planned and find out how to join us</a>.</p>
```

**Comment by calvin-lau-sig7** [comment]
Hi @NoraGDS, would you mind taking a quick look at this update to the homepage 'Whats new'?

File: views/partials/_whats-new.njk [hunk 3]
```diff
@@ -6,7 +6,9 @@
     <div class="govuk-grid-row">
       <div class="govuk-grid-column-two-thirds-from-desktop">
         <h2 id="whats-new" class="govuk-heading-l">What’s new</h2>
-        <p class="govuk-body">9 August 2022: We’ve released GOV.UK Frontend v4.3.0. This feature release includes several ‘quality of life’ improvements to make GOV.UK Frontend work better for service teams and meet their needs.</p>
+<p class="govuk-body">16 August 2022: We're running our flagship online event, Design System Day 2022, on 20 and 21 September.</p>
```
Comment (NoraGDS) [inline views/partials/_whats-new.njk:9]:
```suggestion
<p class="govuk-body">17 August 2022: We're running our flagship online event, Design System Day 2022, on 20 and 21 September.</p>
```

**Review by NoraGDS — APPROVED** [review]
LGTM. Only a small date change

File: views/partials/_whats-new.njk [hunk 4]
```diff
@@ -6,7 +6,9 @@
     <div class="govuk-grid-row">
       <div class="govuk-grid-column-two-thirds-from-desktop">
         <h2 id="whats-new" class="govuk-heading-l">What’s new</h2>
-        <p class="govuk-body">9 August 2022: We’ve released GOV.UK Frontend v4.3.0. This feature release includes several ‘quality of life’ improvements to make GOV.UK Frontend work better for service teams and meet their needs.</p>
+<p class="govuk-body">17 August 2022: We're running our flagship online event, Design System Day 2022, on 20 and 21 September.</p>
+<p class="govuk-body"><a href="https://designnotes.blog.gov.uk/2022/08/16/join-us-for-design-system-day-2022/" class="govuk-link">See what we have planned and find out how to join us</a>.</p>
```
Comment (calvin-lau-sig7) [inline views/partials/_whats-new.njk:10]:
```suggestion
<p class="govuk-body"><a href="https://designnotes.blog.gov.uk/2022/08/16/join-us-for-design-system-day-2022/" class="govuk-link">See what we have planned and how to join us</a>.</p>
```

**Review by NoraGDS — APPROVED** [review]
good change.

**Pull request merged**